docs/guide/controller-programming.md

```diff
@@ -480,13 +480,15 @@ Environment variables in this file can contain references to other environment v
 They will be automatically replaced by the actual value already existing in the environment.
 The Webots "runtime.ini" supports 7 sections:
 
-- `[environment variables with relative paths]`
+- `[environment variables with paths]`
 
-    This section should contain only environment variables with relative paths.
-    Paths must be separated by the colon symbol ':' and the separator between
-    directories is the slash symbol '/'. Variables declared in this section will be
+    This section should only contain environment variables with either relative or
+    absolute paths. Paths must be separated using the colon symbol ':' and the separator for
+    directories to use is the slash symbol '/'. Variables declared in this section will be
     added on every platform. On Windows, colons will be replaced by semicolon and
-    slash will be replaced by backslash according to the Windows syntax.
+    slash will be replaced by backslash according to the Windows syntax.<br>
```

Comment:
We should never use `<br>` to add empty lines, instead, you can enter empty lines, like this:
```suggestion
    slash will be replaced by backslash according to the Windows syntax.

```